inference-engine/include/details/ie_mmap_allocator.hpp

```diff
@@ -0,0 +1,276 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
I think it's better to define two different files for Linux and Windows and use Cmake for appropriate file including. It is very difficult to read file with a lot of ifdef-s.